Geometry/MuonNumbering/interface/DD4hep_RPCNumberingScheme.h

```diff
@@ -0,0 +1,44 @@
+#ifndef GEOMETRY_MUON_NUMBERING_RPC_NUMBERING_SCHEME_H
+#define GEOMETRY_MUON_NUMBERING_RPC_NUMBERING_SCHEME_H
+
+/*
+//\class RPCNumberingScheme
+
+ Description: RPC Numbering Scheme for DD4hep
+              
+//
+// Author:  Sergio Lo Meo (sergio.lo.meo@cern.ch) following what Ianna Osburne made for DTs (DD4HEP migration)
+//          Created:  Fri, 20 Sep 2019 
+*/
+
+#include "Geometry/MuonNumbering/interface/DD4hep_MuonNumbering.h"
+
+class MuonBaseNumber;
+class MuonConstants;
+
+namespace cms {
+  class RPCNumberingScheme {
+  public:
+    RPCNumberingScheme(const MuonConstants& muonConstants);
+
+    void baseNumberToUnitNumber(const MuonBaseNumber&);
+    void SetDetId(int idnew) { detId = idnew; }
```

Comment:
please, use CMS naming conventions: a member function/method name starts with a small letter.
BTW, this method should be private.